### DISCUSSION TITLE
Aligning elements in the group

### DISCUSSION THREAD

**Discussion by Shiraz-V-Kamar** [excalidraw/excalidraw#5051] [General]
At present, when elements are grouped, they do not get affected by any alignment commands like 

- Left align 
- Center align 
- right alien 

Same for the distribution option as well!

I hope to bring this to the notice of the devs!
Thanks a lot for ur hard work 😄 👏🏾 !!

**Comment by EvanNotFound** [comment]
Agree!

**Comment by Mrazator** [comment]
Definitely! Let's track it here #9606
